Sources/Settings/Controller/SettingsController.swift

```diff
@@ -100,7 +98,7 @@ class SettingsController: UITableViewController {
                                              action: #selector(showAbout))
         aboutBarButton.tintColor = PresentationTheme.current.colors.orangeUI
         navigationItem.leftBarButtonItem = aboutBarButton
-        self.navigationItem.leftBarButtonItem?.accessibilityIdentifier = VLCAccessibilityIdentifier.about
```

Comment:
It was strange to call the same `navigationItem` with `self` and without it so close.

Reply:
I can redo it for all class (or all classes) - but it can lead to many merge conflicts if we don't require it in Code Style Guidelines